beaker-vue/src/components/notebook/BeakerNotebook.vue

```diff
@@ -47,7 +47,7 @@ export const BeakerNotebookComponent: DefineComponent<any, any, any>  = defineCo
             if (typeof cell === 'string') {
                 newCellId = cell;
             }
-            else if (Object.hasOwn(cell, "cell")) {
+            else if (Object.prototype.hasOwnProperty.call(cell, "cell")) {
```

Comment:
Ran into a local typescript type-check issue for some reason _only_ on this usage/condition in this file. This works in `dev`. We can look into why `hasOwn` isn't working for this branch now or later. I also tried updating the tsconfig app/lib files to es2022 instead of 2020 to see if that would help. It didn't.

Reply:
Reverting now that we've resolved the package-lock type-check issues.